packages/gestalt/src/Tabs.js

```diff
@@ -25,54 +52,68 @@ export default function Tabs({
     text: React.Node,
   |}>,
   wrap?: boolean,
-|}): React.Node {
-  const handleTabClick = (i: number, e: SyntheticMouseEvent<>) =>
-    onChange({ activeTabIndex: i, event: e });
+|};
 
+export default function Tabs({
+  activeTabIndex,
+  onChange,
+  size = 'md',
+  tabs,
+  wrap,
+}: Props): React.Node {
   return (
-    <div
-      className={classnames(
-        styles.Tabs,
-        wrap && layout.flexWrap,
-        size === 'md' ? layout.medium : layout.large
-      )}
+    <Box
+      alignItems="center"
+      display="flex"
+      marginEnd={-1}
+      marginStart={-1}
+      minHeight={size === 'lg' ? 48 : 40}
       role="tablist"
+      wrap={wrap}
+      dangerouslySetInlineStyle={{
+        __style: { userSelect: 'none' },
+      }}
     >
-      {tabs.map(({ href, id, text }, i) => {
-        const isActive = i === activeTabIndex;
-        const cs = classnames(styles.tab, {
-          [styles.tabIsNotActive]: !isActive,
-          [styles.tabIsActive]: isActive,
-        });
+      {tabs.map(({ id, href, text }, index) => {
+        const isActive = activeTabIndex === index;
         return (
-          <a
-            aria-selected={isActive}
-            className={cs}
-            href={href}
-            {...(id ? { id } : {})}
-            key={`${i}${href}`}
-            onClick={(e: SyntheticMouseEvent<>) => handleTabClick(i, e)}
-            role="tab"
+          <Box
+            key={id || `${href}_${index}`}
+            id={id}
+            dangerouslySetInlineStyle={{
+              __style: { padding: '0px 2px' },
```

Comment:
@jackhsu978 let's not add the padding here, that should be added by the container which uses `Tabs`. Main reason is that <Tabs /> no longer looks visually in sync with other components.

It also makes the assumption that the outline will always stay 2px, which is not the case.

```jsx
<Row padding={2}>
  <Tabs />
</Row>
```

![image](https://user-images.githubusercontent.com/127199/86024104-6adefc00-b9e1-11ea-8ed1-19ede5bf0eaa.png)

Reply:
@christianvuerings There are 2 problems with the current Tab:
1. when it's inside of `<Box overflow="auto">`, the blue accessibility outline gets cut off
2. the blue accessibility overlaps with the adjacent tab.

See snapshot below:

<img width="488" alt="Screenshot 2020-06-29 18 11 43" src="https://user-images.githubusercontent.com/5341184/86071405-039e6780-ba34-11ea-9095-461bf4498a81.png">

My idea here is to leave enough vertical & horizontal padding such that the blue accessibility outline does not get cut off and does not overlap with the adjacent tab. See the snapshot below.
<img width="507" alt="Screenshot 2020-06-29 18 09 57" src="https://user-images.githubusercontent.com/5341184/86071452-19ac2800-ba34-11ea-905b-b00f862ff75f.png">

The horizontal padding is set up as follows:

- tabs root element gets 0.5 boint
- each child tab element gets 0.5 boint
```
(1bt)[ TAB 1 ](1bt)[ TAB 2 ](1bt)[ TAB 3 ](1 bt)
```

Also, the `Tab` component now uses the `<Row>` component and takes the `gap` prop. If additional gap is needed, the `gap` prop would be handy.

```
(1bt)[ TAB 1 ](1bt + 2 * gap)[ TAB 2 ](1bt + 2 * gap)[ Tab 3 ](1bt)
```

What do you think?

Reply:
I changed mind. Now by default, there's no padding at all.

If the `Box` wrapping the Tab component has `overflow` prop equal to auto/scroll/hidden, the blue accessibility box shadow gets cropped, but one can fix that by adding `padding={1}` to the `Box` wrapper.

To fix blue accessibility box shadow overlapping the adjacent tab, one can fix that by adding `gap={1}` to the `Tab` component, which works similar to how `Row` works.

Given that there are fixes to both issues, I am happy with this simple solution.